### DISCUSSION TITLE
Filtro por profesor/es

### DISCUSSION THREAD

**Discussion by wareval0** [Open-Source-Uniandes/Mi-Horario-Uniandes#60] [Ideas]
Sería muy bueno también poder filtrar por el profesor o profesores que dictan el curso. Sería un plus saber la calificación que tienen, ya sea recopilando la información de páginas cómo "los estudiantes" o de encuestas oficiales como la "encuesta de cursos y profesores". No es nada fácil, pero sería genial.

**Comment by c4ts0up** [comment]
👌🏽 Tocaría ver qué datos podemos utilizar o no para ordenar a los profesores. @jsurrea sabes si hay datos de profesores disponibles?
